brands/amenity/charging_station.json

```diff
@@ -90,7 +90,17 @@
       "name": "FLO"
     }
   },
-  "amenity/charging_station|Source London": {
+  "amenity/charging_station|Innogy": {
+    "matchNames": ["RWE"],
+    "tags": {
+      "amenity": "charging_station",
+      "brand": "Innogy",
+      "brand:wikidata": "Q2124721",
+      "brand:wikipedia": "de:Innogy",
+      "name": "Innogy"
+    }
+  },
+  "amenity/charging_station|Source-London": {
```

Comment:
@DavidMoraisFerreira did you mean to change the name of Source London as well?

Reply:
Oh my! I did not intend to commit that change, sorry about that! I was trying to figure out why npm was throwing errors. Shall I open a new PR with the fix?

Reply:
I pushed the changes to #3101.